### ISSUE TITLE
Policy/value learning-rate tuples are ignored by Torch A2C and PPO

### ISSUE THREAD

**Issue by KirillHit** [Toni-SM/skrl#451]
### Description

The Torch A2C and PPO configs allow separate policy and value learning rates:

```python
cfg.learning_rate = (1e-4, 1e-3)
```

However, the optimizer uses only `learning_rate[0]` for both models:

```python
torch.optim.Adam(
    itertools.chain(self.policy.parameters(), self.value.parameters()),
    lr=self.cfg.learning_rate[0],
)
```

Therefore, `learning_rate[1]` is ignored. The same applies to the scheduler and its kwargs, where only element `[0]` is used.

For separate models, both tuple elements should be applied to their respective parameter groups.

### What skrl version are you using?

2.0.0

### What ML framework/library version are you using?

PyTorch

### Additional system information

Linux
